controllers/suite_test.go

```diff
@@ -251,6 +253,11 @@ var _ = BeforeSuite(func() {
 
 	Expect(err).NotTo(HaveOccurred())
 
+	CreateNamespaceWithContext(ctx, &appNamespace)
+
+	err = ApplyResources(filepath.Join("..", "examples", "toystore", "toystore.yaml"), k8sClient, appNamespace)
```

Comment:
Today, we probably don't even need to deploy the app. I reckon our tests won't issue requests to the service. In fact, I believe the HTTPRoutes we create in the tests don't even include backendRefs.

In case we _do_ want to keep deploying the app and eventually start linking the test routes to it, by moving the app to its own namespace, we need to make sure routes created in the test namespaces include backendRefs pointing to the service created here in the `appNamespace` namespace.